library/swift/src/Data+Extension.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+extension Data {
+  /// Returns the next integer in the data, using the size of `T`. Nil if the data is too small.
+  ///
+  /// - returns: The next integer in the data, or nil.
+  func nextInteger<T: FixedWidthInteger>() -> T? {
```

Comment:
The docstring and function name sort of lead me to infer that calling this multiple times on a `Data` object should return subsequent integers (with some sort of implicit internal index).

Reply:
Updating to `integer(atIndex:)` 👍